config/jobs/kubernetes/sig-instrumentation/sig-instrumentation-kind-periodics.yml

```diff
@@ -0,0 +1,59 @@
+periodics:
+- interval: 1h
+  cluster: k8s-infra-prow-build
+  name: ci-kubernetes-kind-e2e-structured-logging
+  annotations:
+    testgrid-dashboards: sig-instrumentation-tests, sig-testing-kind
+    testgrid-tab-name: kind-structured-logging
+    description: Smoke tests Kubelet with structured logging enabled using sigs.k8s.io/kind
+    # TODO: anyone else?
+    testgrid-alert-email: bentheelder@google.com
+    testgrid-num-columns-recent: '6'
+    fork-per-release: "true"
+    fork-per-release-periodic-interval: 1h 2h 6h 24h
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  decorate: true
+  decoration_config:
+    timeout: 60m
+  extra_refs:
+  - org: kubernetes
+    repo: kubernetes
+    base_ref: master
+    path_alias: k8s.io/kubernetes
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/krte:v20210311-6badf7a-master
+      command:
+      - wrapper.sh
+      - bash
+      - -c
+      - curl -sSL https://kind.sigs.k8s.io/dl/latest/linux-amd64.tgz | tar xvfz - -C "${PATH%%:*}/" && e2e-k8s.sh
+      env:
+      # this is the key configuration
+      - name: KUBELET_LOG_FORMAT
+        value: structured
+      # TODO: This is a fairly arbitrary test that exercizes actually scheduling some pod(s).
+      # We mostly just want cluster up/down and something that ensures the cluster is in fact up fully.
+      - name: FOCUS
+        value: Job.should.run.a.job.to.completion.when.tasks.sometimes.fail.and.are.locally.restarted
```

Comment:
this may not be the best option, I literally skimmed a conformance dashboard for something that said "yep, this will create some pods and make me satisfied that we have a kubelet running happily"

if we really wanted we could also check the actual format of the logs coming from kubelet pretty easily someday, but that's just going to fail right now anyhow.